environment.yml

```diff
@@ -1,30 +1,39 @@
 #  environment document
 
-name: mrilanding
+name: one-env-to-rule-them-all
 channels:
   - conda-forge
   
 dependencies:
+  - python>=3.11
   - imageio=2.25.0
   - ipympl=0.9.3
-  - jupyter=1.0.0
-  - jupyterlab=3.6.1
+  - jupyter
+  - jupyterlab
+  - kneed
   - matplotlib
-  - pytest<8.1
+  - matplotlib-inline
   - nibabel
-  - notebook=6.4.10
-  - numpy
-  - pandas
+  - notebook
+  - numpy~=1.20
```

Comment:
I think one of the vendored packages had it pinned to 1.20, so I relaxed it, but excluded 2.0 (since that's quite likely to conflict, ABI change and all).

Reply:
What I mean is: did you intend to restrict to 1.20.x only (which I think is what `~=` does)? Because if you only want to exclude 2.0, you could also do `<1.20`. There are a few 1.x versions after 1.20.

Reply:
Ah awesome, thanks, TIL :)